label.py

```diff
@@ -359,8 +360,34 @@ def show_help():
     print('======================================================================================\n')
 
 
+def images_cmp(x, y):
+    img_list_x = x.split('_')
+    img_list_y = y.split('_')
+    if len(img_list_x) != len(img_list_y):
+        return len(img_list_x) - len(img_list_y)
+    for (f1, f2) in zip(img_list_x, img_list_y):
+        if f1 == f2:
+            continue
+        if f1.isdigit():
+            return int(f1) - int(f2)
+
+
+def group_images():
+    sorted_images_to_show = sorted(images_to_show, key=functools.cmp_to_key(images_cmp))
```

Comment:
Change the images_to_show in this function directly, instead of doing it in main